apps/ml/example_data.json

```diff
@@ -0,0 +1,28 @@
+{
+  "ActivityTracking": {
+    "accelerometer": [{
+      "timestamp": "14-04-2022-16-33-21T02",
+      "data": {
+        "x": "",
+        "y": "",
+        "z": ""
+      }
+    }],
+    "gyroscope": [{
+      "timestamp": "14-04-2022-16-33-21T02",
+      "data": {
+        "x": "",
+        "y": "",
+        "z": ""
+      }
+    }],
+    "magnetometer": [{
+      "timestamp": "14-04-2022-16-33-21T02",
+      "data": {
+        "x": "",
+        "y": "",
+        "z": ""
+      }
+    }]
+  }
+}
```

Comment:
If we have all the models typed correctly in pydantic, there is no need for this data file anymore 🚀